src/main.tsx

```diff
@@ -4,7 +4,7 @@ import { RouterProvider } from "react-router-dom";
 
 import { AppNavigation } from "@/routes/app-navigation";
 
-import "./assets/index.css";
+import "./styles/index.css";
```

Comment:
let's try to use this type of import "@/styles/index.css";

Reply:
resolved